test/modeles/pointAcces.spec.js

```diff
@@ -0,0 +1,17 @@
+const expect = require('expect.js');
+
+const PointAcces = require('../../src/modeles/pointAcces');
+
+describe("Un point d'accès", () => {
+  it('connaît sa description', () => {
+    const pointAcces = new PointAcces({
+      description: 'Une description',
+    });
+
+    expect(pointAcces.description).to.equal('Une description');
+  });
+
+  it('donne la liste des noms de ses propriétés', () => {
+    expect(...(PointAcces.nomsDesProprietes())).to.be(...['description']);
```

Comment:
L'utilisation des `...` ici est inappropriée. Le fait qu'on passe plusieurs paramètres à `expect` et à `be` ne veut pas dire que ces paramètres suivants sont comparés. Ainsi :
```javascript
const listeEvaluee = [1, 2, 3];
const listeAttendue = [1, 6, 8];
expect(...listeEvaluee).to.be(...listeAttendue) // retourne OK
```

Je pense que l'appel qui conviendrait ici est :
```javascript
expect(PointAcces.nomsDesProprietes()).to.eql(['description'])
```